### DISCUSSION TITLE
I have to reload all my thermostats after home assistant restart. Is it normal?

### DISCUSSION THREAD

**Discussion by Apkaloo** [jmcollin78/versatile_thermostat#703] [Q&A]
This integration is great and works very nicely. Thanks.

After each restart of home assistant, all my 11 vtherms are disabled. 
I need to go into the Vtherm integration and reload them one by one manually. Thereafter, I have to turn on the heating of each of them and finally reload the scheduler integration because they start in manual mode. 

1- Is it normal?
2- If not, where should I look to investigate the problem? Something I did wrong? 
3- If this is normal, is there a way to automate reloading and restarting of the vtherms?

Thank you for your assistance.

**Comment by jmcollin78** [comment]
Hello @Apkaloo 

No it is not normal. You have to look in the logs. I thingk something is wrong in your environment. See here to activate logs: https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/en/troubleshooting.md#enable-logs-for-the-versatile-thermostat

**Comment by Apkaloo** [comment]
OK, si ce n'est pas normal alors je vais chercher.
Je regarde ce soir , mais je crains de ne pas comprendre les logs... 😐

Merci pour votre aide et pour ce magnifique programme.

______
Denis Jeanteur
89, route des vallées
37510 Ballan-Miré
06 63 95 40 07

**Reply by jmcollin78** [reply]
Just activate the VTherm logs and send it to me around the time you have the issue.

**Comment by pqhf5kd** [comment]
I'm also noticing that my climate entities are set to (I think) Heat and Manual, but not heating if under but close to the target, after a restart. But I have only been using VTherm for less than a day, I will follow this discussion :)

**Comment by Apkaloo** [comment]
I have tried to analyse the logs. In fact, I have several errors but not affecting directly Versatile thermostat. Maybe this impact the overall starting of my home assistant.
As I don't understand well the different issues, I'm trying to solve them by removing some integrations... So far with no effect. I'm not used to debugging, I'm just a user, not a programmer (and that's perhaps why I'm in trouble).
I did a little patch in Node Red to detect the HA restart and then reload the different Vtherm and then reload the schedulers... It works!
But it's only a patch. But this is not a solution. Sorry about that.
I'll further investigate after the holidays :)
Happy Christmas
